source/slang/slang-diagnostic-defs.h

```diff
@@ -700,6 +700,7 @@ DIAGNOSTIC(
     Error,
     argumentExpectedLValue,
     "argument passed to parameter '$0' must be l-value.")
+DIAGNOSTIC(30078, Error, cannotTakeConstantPointers, "Can not take constant pointers")
```

Comment:
I think this can be made more clear?
```suggestion
DIAGNOSTIC(30078, Error, cannotTakeConstantPointers, "Not allowed to take pointer of immutable object")
```